118.md

```diff
@@ -0,0 +1,89 @@
+# NIP-118
+
+## Polls
+
+`draft` `optional`
+
+This NIP defines the event scheme that describe Polls on nostr.
+
+The NIP makes the following tradeoffs.
+
+1. All votes are public.
+2. All responses are editable (latest by timestamp considered last response)
+
+## Events
+
+### Poll Content
+
+The poll event is defined as a `kind:1068` event. Major tags in the poll event are:
+
+- **label**: The accompanying text of the poll.
+- **option**: The option tags contain an OptionId(any alphanumeric) field, followed by an option label field.
```

Comment:
I don't understand why you need the OptionId. The label can be the ID.

Reply:
what if there are two labels with the same name? i think this is an easy way to get past that. I don't like the idea of tallying same labels together.

Reply:
I feel like a label with the same name but different meaning is not a practical option.

Reply:
I agree with @AsaiToshiya. If clients are only showing the name and not the option ID, the same name should mean the same vote. Not having an ID removes the need to disambiguate names when they duplicate.

Reply:
I disagree, the client is not doing anything in this special case.

Its the poll creator that is just having fun, maybe they want to see how the option order affects the poll results, which they should be allowed to do.

Reply:
I agree with abhay-raizada.  I might make a poll that says:  Who will you vote for?  1) Trump, 2) Trump, 3) Trump
and it should collect the three answers separately.